services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_user_services.py

```diff
@@ -223,6 +223,9 @@ async def progress_create_containers(
 
     scheduler_data.dynamic_sidecar.were_containers_created = True
 
+    # NOTE: user services are already in running state, meaning it is safe to pull inputs
+    await sidecars_client.pull_service_input_ports(dynamic_sidecar_endpoint)
```

Comment:
Question: is this connected to the service progress bar?

Reply:
yes, the progress bar now fills up